src/tools/github2gitlab.rst

```diff
@@ -0,0 +1,73 @@
+=====================================
+Transition your Github Repo to Gitlab
+=====================================
+
+Create Gitlab Account
+=====================
+
+..
+  gitlab.com > create account
+
+Project Members
```

Comment:
this is called project members but it talks about git configuration ... confusing.